### ISSUE TITLE
s7-200cn

### ISSUE THREAD

**Issue by whaosoft** [s7connector/s7connector#149]
s7-200cn without ip  , How to use ??

**Comment by BuckarooBanzay** [comment]
What do you mean without ip? How do you communicate with that plc? 🤔

**Comment by whaosoft** [comment]
> 



> What do you mean without ip? How do you communicate with that plc? 🤔

only use rs485

**Comment by BuckarooBanzay** [comment]
Yeah, i don't think that this project will help you: everything here is tcp/ip based, sorry 🤷
